### PR TITLE
Add timestamps to scheduling checks

### PR DESCRIPTION
**Pull request by benjaminjkraft**

### CODE REVIEW

File: esp/public/media/scripts/program/modules/scheduling_checks.jsx [hunk 1]
```diff
@@ -52,24 +58,34 @@ var SchedulingCheck = React.createClass({
...
     var body;
     if (!this.state.open) {
-      body = <div className="placeholder">(click title to open)</div>;
+      body = <div className="placeholder">
+        (loaded {this.state.timestamp}, click title to open)
+      </div>;
     } else if (this.state.failed) {
-      body = <div className="placeholder">loading failed :(</div>;
+      body = <div className="placeholder">
+        loaded {this.state.timestamp}, loading failed :(
```
Comment (luac) [inline esp/public/media/scripts/program/modules/scheduling_checks.jsx]: Seems kind of odd that this one isn't parenthesized and the others are.

**Comment by luac** [comment]
One comment, otherwise looks good to me.

File: esp/public/media/scripts/program/modules/scheduling_checks.jsx (hunk 1 above)
Comment (benjaminjkraft) [inline esp/public/media/scripts/program/modules/scheduling_checks.jsx]: Fair, I can change that.

**Comment by benjaminjkraft** [comment]
Fixed, along with another minor issue I noticed while fixing that.

**Pull request merged**